app/views/layouts/_sidebar_nav.html.erb

```diff
@@ -1,41 +1,25 @@
-<nav class="mb-6" id="sidebar-nav" aria-label="Primary sidebar nav">
-  <% if user_signed_in? %>
-    <a class="crayons-link crayons-link--block mb-2 flex break-word items-start" id="sidebar-profile" href="#">
-      <span class="crayons-avatar crayons-avatar--xl shrink-0 mr-2">
-        <%# That `src` is basically an empty gif - the tiniest possible image - it serves as a placeholder until the proper profile image is loaded. It's better to have this rather than empty `src` because empty src may look off here and there for a split of a second. %>
-        <img src="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=" class="crayons-avatar__image" id="sidebar-profile--avatar" alt="" width="48" height="48">
-      </span>
-
-      <div class="flex-1">
-        <h4 class="fs-base fw-bold" id="sidebar-profile--name"></h4>
-        <small class="fs-s opacity-75" id="sidebar-profile--username"></small>
-      </div>
-    </a>
-  <% end %>
+<nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" id="sidebar-nav" aria-label="Primary sidebar nav">
+  <a href="<%= root_path %>" class="crayons-link crayons-link--block">
+    <%= inline_svg_tag("twemoji/house.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Home") %>
+    Home
+  </a>
   <% unless user_signed_in? %>
     <a href="<%= sign_up_path %>" class="crayons-link crayons-link--block fw-bold">
       <%= inline_svg_tag("twemoji/handshake.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Sign In/Up") %>
       Sign In/Up
     </a>
   <% end %>
-  <% if false # Let's bring it when ready %>
-    <a href="/" class="crayons-link crayons-link--block">
-      <%= inline_svg_tag("twemoji/house.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Home") %>
-      Home
-    </a>
-  <% end %>
-
-  <div class="sidebar-navigation-links">
-    <% NavigationLink.ordered[0..4].each do |link| %>
-      <%= render "articles/sidebar_nav_link", link: link %>
+  <div class="sidebar-navigation-links spec-sidebar-navigation-links">
+    <% NavigationLink.all.ordered[0..4].each do |link| %>
```

Comment:
Same [comment as the hamburger partial]() links below, can we store NavigationLink.all in a variable and then use that variable?

Reply:
I'm not sure if you've seen this earlier but I tried that: https://github.com/forem/forem/pull/10424#discussion_r514061732

Reply:
I tried this locally and it seems to work for me:

```ruby
<% nav_links = NavigationLink.all.ordered.to_a %>

...

<% nav_links[0..4].each do |link| %>

...

<% if nav_links.length > 5 %>

...


<% nav_links[5..].each do |link| %>
```

They are simple once the query is done in the right order, but let me know if you want me to push the changes :)

Reply:
Thanks, this time it works.. Not sure, maybe I did something wrong previously... :D